src/main/java/com/iexec/sms/secret/web3/Web3Secret.java

```diff
@@ -17,29 +17,55 @@
 package com.iexec.sms.secret.web3;
 
 import com.iexec.sms.secret.Secret;
-import lombok.Data;
-import lombok.EqualsAndHashCode;
-import lombok.Getter;
-import lombok.NoArgsConstructor;
+import lombok.*;
```

Comment:
Same remark on imports here

Reply:
Fixed in https://github.com/iExecBlockchainComputing/iexec-sms/pull/134/commits/12e34cd7987a287063d3eb6bc730db7d0f69d0d1.